packages/termux-tools/pkg

```diff
@@ -80,6 +82,7 @@ case "$CMD" in
 		update_apt_cache
 		apt install "$@"
 		;;
+	autoclean) apt autoclean;;
```

Comment:
maybe to match all the others it should be just au*, or autoc*? maybe ac|au*?

Reply:
Nah, I don't think it's a good idea to provide such shortcuts, as that makes it tough to add commands later without changing the shortcuts' behavior.

Reply:
You need to either follow the general style of option definitions or make all options full-named.

Reply:
This has nothing to do with "style," but with letting users type the shorter `pkg se xml` and have it search for packages. By allowing so many shortcuts, you make it harder to add commands later: I see no value in letting the user type `pkg autoc` instead of the full `pkg autoclean`.